### DISCUSSION TITLE
Make bCNC a comunity project

### DISCUSSION THREAD

**Discussion by m1ch** [vlachoudis/bCNC#1718] [Ideas]
bCNC is a great piece of software, but as far as I can tell it relies only on @vlachoudis. I would like to contribute to the project, but as long as it relies on only one person to merge all pull-requests, I fear that it will be slow progress. 

My suggestion to @vlachoudis to move bCNC to a bCNC organization and add some deserving members as collaborator, so they can also merge pull requests. 

@Harvie @bosd @LittlePierre @rschell as you are some resent contributors, please add your opinion.

**Comment by Harvie** [comment]
Actually i can merge PRs. But i have to review them first to make sure they make sense to me.

**Comment by bosd** [comment]
Don't have a very strong opinion on this. But in many open source projects development is slow, because of the lack of capacity of the maintainers for reviewing and merging pr's.

Did'nt make a lot of PR's here yet, but the experience until now, it has been quick :)

> Actually i can merge PRs. 
So we have 2 maintainers on this repo?
If needed, can more maintainers be added??

This led me to the question, what is the difference between a private repo and an organization repo?
Could't find a quick answer, with an comparison chart.
At first glance, it looks like a private repo also can have multiple collaborators and roles.

it's a different topic. But I would like to have some sort of a dev roadmap for bCNC.
